test/TestExtensions/API.Test/InternalAPITestHook.cs

```diff
@@ -6,6 +7,18 @@ namespace API.Test
 {
     public static class InternalAPITestHook
     {
+        public static void InstallLatestPackageApi(string id, bool prerelease)
+        {
+            var dte = ServiceLocator.GetInstance<EnvDTE.DTE>();
+            var services = ServiceLocator.GetInstance<IVsPackageInstaller2>();
```

Comment:
or it's an earlier version that doesn't provide this service yet.
Given the scope of e2e tests I guess we will never have incompatible versions.